Test.groovy

```diff
@@ -79,5 +79,18 @@ pipeline {
       sh "make down"
       junit "**/test-reports/*.xml"
     }
+    success {
+      build job: 'specification', parameters: [
+        string(name: 'BRANCH', value: GIT_BRANCH.split('/')[1])
```

Comment:
I don't know anything about jenkins. But I wonder if the index of `[1]` here is really correct ... or maybe it's also the `TAG` ?

Reply:
GIT_BRANCH has the shape origin/my-feature-branch.